src/auto-evo/simulation/SimulationCache.cs

```diff
@@ -431,6 +442,9 @@ public void Clear()
         slimeJetScore *= MathF.Sqrt(slimeJetsCount);
         slimeJetScore *= mucilageMultiplier;
 
+        mucocystsScore *= MathF.Sqrt(mucocystsCount);
+        slimeJetScore += mucocystsScore;
+
         var predationToolsRawScores = (pilusScore, oxytoxyScore, slimeJetScore);
```

Comment:
Mucocysts really should be their own "predation tool" instead of adding to slime jet, as they do very different things.